docs/intro/README.md

```diff
@@ -0,0 +1,35 @@
+# Overview
+
+Regen Ledger is a public Proof-of-Stake (PoS) blockchain application built with [Cosmos SDK](https://docs.cosmos.network/). Regen Ledger provides the infrastructure for a public ecological accounting system for verification of claims, agreements, and data related to ecological state.
+
+## Regen Mainnet
+
+[Regen Mainnet](https://www.regen.network/mainnet/) is the primary network supported by Regen Ledger. When referring to the Regen Network blockchain, we are referring to Regen Mainnet. Regen Mainnet is powered by the official [REGEN](https://www.regen.network/token/) token and there are currently 75 validators securing the network. 
+
+For more information, check out [Getting Started: Live Networks](../getting-started/live-networks).
+
+## Regen Devnet
+
+Regen Devnet is a test network available to developers who would like to test out features of Regen Ledger in a live network scenario. Unofficial REGEN tokens can be redeemed by anyone using a faucet, enabling developers to test out experimental features not yet available on Regen Mainnet.
+
+For more information, check out [Getting Started: Live Networks](../getting-started/live-networks).
```

Comment:
This will need to be updated to reflect the two testnets that are user focused. I think it makes sense to leave this as is for now and then to update in another pull request once we have the test networks set up.

Reply:
agreed.